### PR TITLE
[Doc] add installation instruction

### LINKED ISSUE #36: Tests Documentation
- [x] Add instructions for running the tests on the documentation
- [x] Add instructions for running the tests on the main README

### PR DESCRIPTION
**Pull request by skoudoro**
Quick update of installation instruction and community page

fix #36

### CODE REVIEW

File: docs/source/installation.rst [hunk 1]
```diff
@@ -2,6 +2,78 @@
 Installation
 ============
 
-At the command line::
+FURY supports Python 3.5+. You can currently still use Python 2.7 but it will be removed as the Python 2.7 end of life
+is on December 2019, 31rst
```
Comment (Garyfallidis) [inline docs/source/installation.rst]: 31?

File: docs/source/installation.rst [hunk 2]
```diff
@@ -2,6 +2,78 @@
...
+**Step 4:** Enjoy!
+
+Test the Installation
+---------------------
+
+You can check your installation via this command::
+
+    $ python -c "from fury import get_info; print(get_info())"
+
+It should return all the informations about FURY. After this success, we recommend you to play with all tutorial!
```
Comment (Garyfallidis) [inline docs/source/installation.rst]: bad english

File: docs/source/installation.rst [hunk 3]
```diff
@@ -2,6 +2,78 @@
...
+
+Running the Tests
+-----------------
+
+There is two ways to run FURY tests:
```
Comment (Garyfallidis) [inline docs/source/installation.rst]: There are

File: docs/source/installation.rst [hunk 4]
```diff
@@ -2,6 +2,78 @@
...
+
+- From python interpreter::
+
+    $ from fury.tests import test
+    $ test()
+
+- From the command line. You need to be on FURY package folder::
```
Comment (Garyfallidis) [inline docs/source/installation.rst]: on the

File: docs/source/installation.rst [hunk 5]
```diff
@@ -3,7 +3,7 @@ Installation
 ============
 
 FURY supports Python 3.5+. You can currently still use Python 2.7 but it will be removed as the Python 2.7 end of life
-is on December 2019, 31rst
+is on December 2019, 31
```
Comment (Garyfallidis) [inline docs/source/installation.rst]: December 31st 2019

File: docs/source/installation.rst [hunk 6]
```diff
@@ -62,18 +62,18 @@ You can check your installation via this command::
 
     $ python -c "from fury import get_info; print(get_info())"
 
-It should return all the informations about FURY. After this success, we recommend you to play with all tutorial!
+This command will give you all informations about FURY. The next step will be to run a :doc:`Tutorial </tutorial/index>`.
```
Comment (Garyfallidis) [inline docs/source/installation.rst]: will give you important information about FURY's installation.

File: docs/source/community.rst [hunk 7]
```diff
@@ -1,6 +1,16 @@
...
+=========
+Community
+=========
+
+Join Us!
+--------
+
+- via Slack
+- via mailing list
+- via github
```
Comment (arokem) [inline docs/source/community.rst]: Add links?

File: docs/source/installation.rst [hunk 8]
```diff
@@ -2,6 +2,78 @@
...
+The optional dependencies are:
+
+- matplotlib >= 2.0.0
+- dipy >= 0.16.0
+
+
+Installation with PyPi
+----------------------
+
+Just use the `pip` command line::
```
Comment (arokem) [inline docs/source/installation.rst]:
```suggestion
In a terminal, issue the following command::
```

Comment (arokem) [reply docs/source/installation.rst]: The word "just" is problematic: https://www.excitant.co.uk/just-is-a-dangerous-word/

Comment (skoudoro) [reply docs/source/installation.rst]: Ahaha, I read the opposite on a LinkedIn article 😄 . But I prefer your sentence. (I will find this article)

File: docs/source/community.rst (hunk 7 above)
Comment (skoudoro) [reply docs/source/community.rst]: yes!

**Pull request merged**